addons/test_discuss_full/tests/test_performance.py

```diff
@@ -142,7 +142,7 @@ def _get_init_messaging_result(self):
                         'custom_channel_name': False,
                         'id': self.channel_general.id,
                         'memberCount': len(self.group_user.users | self.user_root),
-                        'serverMessageUnreadCounter': 5,
```

Comment:
This rename could be merged completely separately to ease the diff a little?

Reply:
It's all in the 1st commit, which is much easier to digest